i18n/locales/en.json

```
@@ -57,6 +57,7 @@
   "expandable_content_show_less_button_label": "Show Less",
   "expandable_content_show_more_button_label": "Show More",
   "footer_contact_us": "Contact Us",
+  "footer_license": "Licensed in GPL v3",
```

GPLv3 seems one word 
https://www.gnu.org/licenses/quick-guide-gplv3.html
```suggestion
  "footer_license": "Licensed in GPLv3",
```